src/c_frontend/tdi_table_c.cpp

```diff
@@ -408,122 +395,23 @@ tdi_status_t tdi_table_action_data_allocate_container_with_fields(
 
 #ifdef _TDI_FROM_BFRT
```

Comment:
Do we enable the table tdi_table_entry_scope_attributes_allocate now? If we do, please remote the _TDI_FROM_BFRT.

Reply:
No, the c frontend work for attributes is still remaining. We need to add that support once the separation of logic in the CLI from common open source to target specific code is done.